intel_extension_for_transformers/neural_chat/examples/serving/README.md

```diff
@@ -0,0 +1,62 @@
+# Serving NeuralChat Text Generation with Triton Inference Server
```

Comment:
Shall we enable chatbot by leveraging the triton inference server?

Reply:
Yes, inside a docker container a textchat chatbot is built and and call `chatbot.predict` when a request is coming. Triton Inference Server itself is responsible for receiving and dispatching the requests for us and a chatbot will do the inference.